src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,188 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+@TestMethodOrder(OrderAnnotation.class)
+public class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private static final String loginNull = null;
+    private static final String loginDefault = "login";
+    private static final String passNull = null;
+    private static final String passShort = "short";
+    private static final String passDefault = "password";
+    private static final Integer ageNull = null;
+    private static final Integer ageYoung = 17;
+    private static final Integer ageBarely = 18;
+    private static final Integer ageMax = Integer.MAX_VALUE;
+    private static final Integer ageMin = Integer.MIN_VALUE;
+    private static final Integer ageDefault = 25;
```

Comment:
I dont think we need this variables at class level, and we really don't need any individual variables for null values